packages/bot-web-ui/src/pages/bot-builder/toolbar/__tests__/toolbar.spec.tsx

```diff
@@ -81,4 +82,16 @@ describe('Toolbar component', () => {
         render(<Toolbar />);
         expect(await screen.findByRole('button')).toBeInTheDocument();
     });
+
+    it('Toolbar should render Quick strategy form on toolbar button click and when it is mobile version', async () => {
+        (useStore as jest.Mock).mockReturnValue({
+            ui: {
+                is_mobile: true,
+            },
+        });
+        render(<Toolbar />);
+        const quick_strategy_button = screen.getByText('Quick strategy');
+        userEvent.click(quick_strategy_button);
```

Comment:
as we are using async let's put await before userEvent

Reply:
alright